splunk-devops-extend/src/main/java/com/splunk/splunkjenkins/console/LabelConsoleLineStream.java

```diff
@@ -45,7 +45,7 @@ protected void eol() {
         String line = decoder.decodeLine(branch.getBuffer(), branch.size());
         // reuse the buffer under normal circumstances
         branch.reset();
-        if (line != null && !"".equals(line)) {
+        if (line != null && !"".equals(line.trim())) {
```

Comment:
could you please change it to `StringUtils.isNotBlank(line)` (self explaining) and update commit accordingly?  
trim() looks for spaces for start&end positions,  for strings like 
`white-spaces        a-long-line   white-spaces` , it may have performance impact than isNotBlank

Reply:
trim creates a new string object which we don't use except for comparing

Reply:
Updated comparison to use StringUtils.isnotBlank().